c2cgeoportal/scripts/c2ctool.py

```diff
@@ -126,6 +126,17 @@ def _fill_arguments(command):
         help="Specify the remote branch",
         default="origin",
     )
+    parser.add_argument(
+        "--index-url",
+        help="An alternate camptocamp python package index URL",
+        default="http://pypi.camptocamp.net/internal-pypi/index/c2cgeoportal"
+    )
+    parser.add_argument(
+        "--c2cgeoportal-url",
+        help="An alternate c2cgeoportal egg/wheel URL (with arguments %(package) %(version)s)",
+        default="http://pypi.camptocamp.net/internal-pypi/index/%(package)s-%(version)s.tar.gz"
+    )
```

Comment:
I know it's related to upgrading to 2.0 but I fail to understand why users would need to use an "alternate Camptocamp python package index". I don't even know what an "alternate Camptocamp python package index" is to be honest :)